app/models/stache.rb

```diff
@@ -3,4 +3,19 @@ class Stache < ActiveRecord::Base
   has_many :categories, through: :stache_categories
   has_many :order_staches
   has_many :orders, through: :order_staches
+  validates :name, presence: true, uniqueness: true
+  validates :description, presence: true
+  validates :price, presence: true, numericality: {
+                   greater_than_or_equal_to: 0.0}
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Space inside } missing.